app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -50,6 +49,10 @@ class LuciBuildService {
 
   static const Set<Status> failStatusSet = <Status>{Status.canceled, Status.failure, Status.infraFailure};
 
+  static const int kDefaultPriority = 30;
+  static const int kRerunPriority = 29;
+  static const int kReleasePriority = 25;
```

Comment:
We agreed to use same priority for release builds?

Reply:
This is for their presubmits. When we start rolling out the cocoon scheduler on release branches, we'll need to revisit this. I filed https://github.com/flutter/flutter/issues/99876 to follow up

Reply:
`kReleasePriority` is only referenced in post submit function `_createPostsubmitScheduleBuild` below, but is not injected to ScheduleBuildRequest. I am a bit confused about how this is related to the pre-submit schedule and which logic in this PR is expected to use it.

Reply:
Ah, you're right. I removed this logic (I'm not sure why we had it there in the first place)